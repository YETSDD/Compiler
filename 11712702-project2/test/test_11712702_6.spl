struct Apple
{
  int weight;
  float round;
};
struct Banana{
  int weight1;
  float round1;
};
int test_11712702_1()
{
  struct Apple aa;
  struct Banana bb;
  float weight_test = 1.0;

  return aa[0];
}
