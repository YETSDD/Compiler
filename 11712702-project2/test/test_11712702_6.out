Error type 10 at Line 16: apply indexing operator to non-array variable
Error type 8 at Line 16: incompatiable return type
